core/server/services/auth/authorize.js

```diff
@@ -35,6 +36,12 @@ authorize = {
 
             return next();
         };
+    },
+
+    authorizeAdminAPI: [session.ensureUser],
+
+    authorizeContentAPI: (req, res, next) => {
```

Comment:
Was wondering why this is empty. Is that part of another PR?
If so, i'd like to suggest to not touch the public auth logic at all? But let me know :)

Reply:
It's a place holder yeah - idea was could wire up contentAPI to it now, and then can be developed on until this is "filled in". Will remove if you prefer :+1:

Reply:
> Will remove if you prefer 👍

👍